### ISSUE TITLE
IE Issues : Column width is Changing Automatically While Scorlling

### ISSUE THREAD

**Issue by vikashshukla** [mkoryak/floatThead#209]
Hi,

I am facing problem in Internet Explorer while scrolling Column width is Changing Automatically.
Problem is only in IE.
I have also attach screen shot as attachment.

http://jsfiddle.net/VikashPlanetreUS/dFp6f/47/

thanks in advance.

![beforescorlling](https://cloud.githubusercontent.com/assets/8761223/7880489/8120b6ee-0617-11e5-931c-cbacf39892cb.PNG)
![afterscrolling](https://cloud.githubusercontent.com/assets/8761223/7880492/8b7203d2-0617-11e5-9b1e-d2494209a530.PNG)

**Comment by mkoryak** [comment]
you should check out the faq in the docs. inline styles can cause issues because I do not copy styles and rely on you to style by class.

Do you really need to use a `colgroup` here? I ask because not many people know about them or use them

Also, this html is a bit hard to follow, if you can clean it to a point where the issue still exists but its not impossible to follow, i can take a look at it some more

**Comment by vikashshukla** [comment]
Thanks for quick response.
'useAbsolutePositioning = false' working fine for me.
